build.gradle

```diff
@@ -222,15 +222,25 @@ allprojects {
             endWithNewline()
         }
         java {
-            target 'checker/**/*.java',
-                'checker-qual/**/*.java',
-                'checker-util/**/*.java',
-                'dataflow/**/*.java',
-                'docs/examples/**/*.java',
-                'docs/tutorial/**/*.java',
-                'framework/**/*.java',
-                'framework-test/**/*.java',
-                'javacutil/**/*.java'
+            def targets = [
+                    // add target folders here
+                    'checker',
+                    'checker-qual',
+                    'checker-util',
+                    'dataflow',
+                    'docs/examples',
+                    'docs/tutorial',
+                    'framework',
+                    'framework-test',
+                    'javacutil',
+            ]
+            targets = targets.collectMany {[
+                    // must call toString() to convert GString to String
+                    "${it}/**/*.java".toString(),
+                    "${it}/**/*.ajava".toString()
+            ]}
+            target targets
```

Comment:
Should the `// TODO: Formatting of this` a few lines down be removed now?

Reply:
Yes, it's removed now.